### PR TITLE
Add findplacebytext method and fields params

### PR DESCRIPTION
**Pull request by stephenmcd**

### CODE REVIEW

File: lib/apis/places.js [hunk 1]
```diff
@@ -18,6 +18,38 @@
...
+    inputtype: v.oneOf(['textquery', 'phonenumber']),
+    language: v.optional(v.string),
+    fields: v.optional(utils.arrayOf(v.oneOf([
+      'alt_id', 'formatted_address', 'geometry', 'icon', 'id', 'name',
+      'permanently_closed', 'photos', 'place_id', 'scope', 'types',
+      'vicinity', 'opening_hours', 'price_level', 'rating'
+    ]), ',')),
+    retryOptions: v.optional(utils.retryOptions),
+    timeout: v.optional(v.number)
+  })
```
Comment (nvinnik) [inline lib/apis/places.js]: we also support locationbias paramater https://developers.google.com/places/web-service/search#FindPlaceRequests

Comment (stephenmcd) [reply lib/apis/places.js]: Thanks, that's added now in bab91d76748b207f3614bed991d071af1d28fced

File: lib/apis/places.js [hunk 2]
```diff
@@ -18,6 +18,50 @@
...
+ */
+exports.findPlaces = {
+  url: 'https://maps.googleapis.com/maps/api/place/findplacefromtext/json',
+  validator: v.compose([
+    v.object({
+      input: v.string,
+      inputtype: v.oneOf(['textquery', 'phonenumber']),
+      language: v.optional(v.string),
+      fields: v.optional(utils.arrayOf(v.oneOf([
+        'alt_id', 'formatted_address', 'geometry', 'icon', 'id', 'name',
```
Comment (nvinnik) [inline lib/apis/places.js]: can we remove alt_id? I dont think this API supports it.

Comment (stephenmcd) [reply lib/apis/places.js]: Done in e0956a9e591fbbd30e86b8d53cb0ac3cc5248ed2

**Pull request merged**